src/check_bgp_protocols.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+"""InnoGames Monitoring Plugins - BIRD BGP Protocols Health Check
+
+This is a Nagios script which queries BIRD to check if all BGP protocols
+are up and with established connections.
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import os
+import re
+
+from argparse import ArgumentParser
+from subprocess import Popen, PIPE, STDOUT
+
+
+def parse_args():
+    parser = ArgumentParser(
+        description='Check if all BIRD BGP protocols are up and established'
+    )
+    parser.add_argument(
+        'path',
+        type=str,
+        help='Path to the birdc/birdc6 binary'
+    )
```

Comment:
Without `type=str` I guess it will fit in one line.

```suggestion
    parser.add_argument('path', help='Path to the birdc/birdc6 binary')
```